### DISCUSSION TITLE
`ConfigProvider` component warning log in Nuxt v3

### DISCUSSION THREAD

**Discussion by dungsil** [unovue/reka-ui#1269] [Q&A]
Hello, a Warning message is displayed when using `ConfigProvider` in Nuxt 3. Is there something I have configured incorrectly?

``` [Vue warn]: Extraneous non-props attributes (data-v-inspector) were passed to component but could not be automatically inherited because component renders fragment or text root nodes.```

```vue
// app.vue

<template>
  <r-config-provider :dir="currentDirection" :use-id="() => useId()">
    <nuxt-route-announcer />
    <nuxt-loading-indicator />

    <nuxt-layout>
      <nuxt-page />
    </nuxt-layout>
  </r-config-provider>
</template>
```

**Comment by frandmb** [comment]
https://github.com/nuxt/devtools/issues/722
